src/constants/winnings.js

```diff
@@ -0,0 +1,7 @@
+export const RANK_INFO = [
+  { name: 'fifth', matchCount: 3, money: 5000 },
+  { name: 'fourth', matchCount: 4, money: 50000 },
+  { name: 'third', matchCount: 5, money: 1500000 },
+  { name: 'second', matchCount: 5, money: 30000000 },
+  { name: 'first', matchCount: 6, money: 2000000000 },
```

Comment:
큰 숫자를 사용할 때는 단위를 쉽게 파악할 수 있도록 numeric sperator의 사용을 고려해보셔도 좋을거 같습니다!
```suggestion
  { name: 'first', matchCount: 6, money: 2_000_000_000 },
```

Reply:
오 너무 좋은 기능 같습니다. javascript 자체 기능이네요!

[796bba8](https://github.com/next-step/js-lotto/pull/324/commits/796bba8906641177dfd7e9b0e5c301bd522a5699)